### ISSUE TITLE
Implement usable_once flag

### ISSUE THREAD

**Issue by lillian-lemmer** [lillian-lemmer/hypatia#59]
Create `usable_once` property for NPCs in NPC.ini and tmx whereas an NPC is destroyed after it is "used." A problem I see is what constitutes "used?"

**Comment by ejmr** [comment]
There will probably be multiple useful definitions of "used" in this context.  One I can think of is that we can consider an NPC "used" once it reaches a certain position, i.e. its X-Y coordinates equal certain values in the configuration which mean, "When the NPC gets to here, the game can remove it."

**Comment by lillian-lemmer** [comment]
Yeah for now I was just thinking after it's been "activated" so i can make doors :)

**Comment by ejmr** [comment]
On that note, what exactly is the purpose of the `after_activate=destroy` setting for the door in the `NPC.ini` file in the demo?

**Comment by lillian-lemmer** [comment]
[Dragon Warrior (Game Boy Color) style door interaction](https://youtu.be/SFKWxhLGJUA?t=5m1s).

**Comment by ejmr** [comment]
Talk about nostalgia.  Dragon Warrior on the NES was the first RPG I ever played, heh.

**Comment by ejmr** [comment]
See commit bcd9745092751e855c29614bd4521e3ce5d8ac26 for an example of how these changes can be used to create an NPC representing a door which we could destroy (or do whatever else) based on its `active` status.

**Comment by ejmr** [comment]
2d406f71675764095fc08895ce958811dec67eb0 fails on Travis-CI due to a PEP8 style error in `hypatia/player.py`.  However, running the test suite locally does not produce this error, i.e. the file passes PEP8 cleanly.  Not sure what the cause may be---difference in configuration between Travis-CI and our local `test.sh`?

**Comment by lillian-lemmer** [comment]
I'll try looking it up in a bit. :) I'll have to review the changes.
